app/src/main/res/values/strings.xml

```diff
@@ -1382,4 +1382,13 @@
     <string name="cannula">Cannula</string>
     <string name="userentry">User entry</string>
 
+    <string name="nav_themeselector">Select Color</string>
+    <string name="darkmode">Dark mode</string>
+    <string name="background_color">Background Color</string>
+    <string name="color_dark">Color dark</string>
+    <string name="color_light">Color light</string>
+    <string name="color_default">Default</string>
+    <string name="color_selected">Selected</string>
+    <string name="theme_switcher_summary">In order to take effect please restart the app.</string>
```

Comment:
This copy is from before we were able to trigger the switch while the app was running.

Reply:
done